packages/cloud-cognitive/src/components/NonLinearReading/NonLinearReading.stories.js

```diff
@@ -26,31 +27,77 @@ export default {
       page: mdx,
     },
   },
+  argTypes: {
+    definition: {
+      table: {
+        disable: true,
+      },
+    },
+    theme: {
+      control: { type: null },
+    },
```

Comment:
When using `getSelectedCarbonTheme`, hide the theme radio toggle feature in Storybook.